File: Examples/AspNetMvcCS/Controllers/DocumentViewerController.Highlight.cs
using System.Web.Mvc;
using GleamTech.DocumentUltimate.AspNet.UI;

namespace GleamTech.DocumentUltimateExamples.AspNetMvcCS.Controllers
{
    public partial class DocumentViewerController
    {
        public ActionResult Highlight()
        {
            var documentViewer = new DocumentViewer
            {
                Width = 960,
                Height = 720,
                Resizable = true,
                Document = "~/App_Data/ExampleFiles/Default.doc",
                SearchOptions =
                {
                    Term = "ancient mariner",
                    MatchOptions = DocumentViewerMatchOptions.MatchAnyWord
                }
            };

            return View(documentViewer);
        }
    }
}

File: Examples/AspNetMvcVB/Controllers/DocumentViewerController.Highlight.vb
Imports GleamTech.DocumentUltimate.AspNet.UI

Namespace Controllers
    Partial Public Class  DocumentViewerController
        Inherits Controller

    Public Function Highlight() As ActionResult
		Dim documentViewer = New DocumentViewer() With {
			.Width = 800,
			.Height = 600,
			.Resizable = True,
			.Document = "~/App_Data/ExampleFiles/Default.doc",
			.SearchOptions = New DocumentViewerSearchOptions With
		        {
		            .Term = "ancient mariner",
		            .MatchOptions = DocumentViewerMatchOptions.MatchAnyWord
		        }
		}

		Return View(documentViewer)
	End Function

    End Class
End Namespace
